src/NServiceBus.RavenDB.ClusterWide.AcceptanceTests/packages.config

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
What is this config doing here :D? This should probably be turned into a package reference

Reply:
I think this was added by adding the new project, good catch!